src/app/sharedUI/screens/Artwork/ArtworkWebView.tsx

```diff
@@ -21,7 +21,7 @@ export const ArtworkWebView = () => {
         onLoad={() => setIsProgressBarVisible(false)}
       />
       {isProgressBarVisible && (
-        <Flex px={4} height="100%" justifyContent="center">
+        <Flex px={4} height="100%">
```

Comment:
As per @brainbicycle's comment, if it has to match with Eigen, then we dont need padding at all
```suggestion
        <Flex height="100%">
```